src/store/index.ts

```diff
@@ -1,11 +1,47 @@
 import Vue from "vue";
 import Vuex from "vuex";
+// import Language from "@/store/modules/Language";
```

Comment:
How do we load this module?

Reply:
Using dynamic : true, Vuex allows us to register modules into store at runtime after store is constructed